### PR TITLE
example: add dnx script

### PR DESCRIPTION
**Pull request by axetroy**
Similar to [npx](https://github.com/npm/npx) example.

execute script and then clean up your disk.

### CODE REVIEW

**Comment by axetroy** [comment]
截屏2019-11-1018 21 11


it seems a problme with network connection in CI

File: std/examples/denox.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```
Comment (ry) [inline std/examples/denox.ts]: dnx.ts ?

Comment (kevinkassimo) [reply std/examples/denox.ts]: How about `dex`? (DEno + EXecute)

Comment (axetroy) [reply std/examples/denox.ts]: rename to dnx.ts now.

**Comment by sholladay** [comment]
> It will delete the downloaded script resource after the process exits.

So this is basically `deno run` but it uses a separate module cache that is deleted afterwards?

**Comment by axetroy** [comment]
> > It will delete the downloaded script resource after the process exits.
> 
> So this is basically `deno run` but it uses a separate module cache that is deleted afterwards?

Exactly!

**Pull request closed without merging**